src/main/kotlin/caculator/ui/Input.kt

```diff
@@ -0,0 +1,53 @@
+package caculator.ui
+
+class Input(inputString: String) {
+    val numbers: List<Int>
+
+    init {
+        val result = Regex("//(.)\n(.*)").find(inputString)
+        numbers = result?.let {
+            splitInputString(it.groupValues[2], Regex(it.groupValues[1]))
```

Comment:
[destructured](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/-match-result/destructured.html)를 이용하면 아래와 같이도 사용이 가능해요!!

```suggestion
            val (customDelimeter, inputString) = it.destructured
            splitInputString(inputString, Regex(customDelimeter))
```